app/presenters/research_output_presenter.rb

```diff
@@ -11,13 +11,13 @@ def initialize(research_output:)
   # Returns the output_type list for a select_tag
   def selectable_output_types
     ResearchOutput.output_types
-                  .map { |k, _v| [k.humanize, k] }
+                  .map { |k, _v| [_(k.humanize), k] }
```

Comment:
This wont work as the key is not yet known when the translation rake task analyses the code. We should set it back to ```.map { |k, _v| [k.humanize, k] }``` to avoid confusion.

same as the change on line 20.

Reply:
I think we might be ok here because of the way we are listing the strings of the enums in `_research_output.erb`.
I added a couple of translations in dmp-assistant-test. "Audiovisual" is an entry for `ResearchOutput.output_types` and "Restricted" is an entry for `ResearchOutput.accesses` (they both got exported to translation.io after being defined in `_research_output.erb`). The translations are looking ok in the dropdowns.

<img width="787" alt="Screenshot 2023-08-01 at 3 22 32 PM" src="https://github.com/portagenetwork/roadmap/assets/71047780/49459792-436a-4542-8700-b063ec451f7b">


<img width="317" alt="Screenshot 2023-08-01 at 3 21 37 PM" src="https://github.com/portagenetwork/roadmap/assets/71047780/5dd98648-df50-4608-a407-eae4e81c9748">


<img width="256" alt="Screenshot 2023-08-01 at 3 21 55 PM" src="https://github.com/portagenetwork/roadmap/assets/71047780/9af6718c-5b9d-492c-be86-e0c0ad6acba2">

Reply:
Or sorry, maybe i'm misunderstanding your concern. If a translation exists, then `_(k.humanize)` will provide the translation. However, if `k.humanize` is not already exported to translation.io, then `_(k.humanize)` cannot help us.